test/Ben.Demystifier.Test/MethodTests.cs

```diff
@@ -77,11 +77,18 @@ public void DemistifiesMethodWithLambda()
             stackTrace = LineEndingsHelper.RemoveLineEndings(stackTrace);
             var trace = string.Join(string.Empty, stackTrace.Split(new[] { Environment.NewLine }, StringSplitOptions.RemoveEmptyEntries));
 
+#if (NETFRAMEWORK && !DEBUG)
```

Comment:
This appears unrelated to the changes in this PR btw... I suspect something obscure changed in the .NET Framework such that this is now required.